pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py

```diff
@@ -29,7 +29,8 @@
 _TEST_DATA_FILE_NAME = RosPack().get_path("pilz_industrial_motion_testutils") + "/test_data/testdata_deprecated.xml"
 PLANNING_GROUP_NAME = "manipulator"
 TARGET_LINK_NAME = "prbt_tcp"
-API_VERSION = "1"
+BASE_LINK_NAME = "prbt_base"
+API_VERSION = "2"
```

Comment:
I would like to open the discussion if we support backwards compatibility to some degree...
Any opinion? To much effort?

Reply:
+1 for allowing execution of API=1 programs (with former behaviour)